pkg/disk/nodeserver.go

```diff
@@ -611,18 +611,22 @@ func (ns *nodeServer) NodeStageVolume(ctx context.Context, req *csi.NodeStageVol
 
 	// sysConfig
 	if value, ok := req.VolumeContext[SysConfigTag]; ok {
-		configList := strings.Split(strings.TrimSpace(value), ",")
-		for _, configStr := range configList {
-			key, value, found := strings.Cut(configStr, "=")
-			if !found {
-				log.Errorf("NodeStageVolume: Volume Block System Config with format error: %s", configStr)
-				return nil, status.Error(defaultErrCode, "NodeStageVolume: Volume Block System Config with format error "+configStr)
+		sysConfigs, err := utilsio.ParseSysConfigs(value, func(key string) bool {
+			return strings.HasPrefix(key, "queue/")
+		})
+		if err != nil {
+			return nil, status.Error(codes.InvalidArgument, err.Error())
```

Comment:
Should we also add this validation to CreateVolume? I think we should report errors as early as possible.